README.md

```diff
@@ -130,6 +130,12 @@ logging:
   # regardless of the configured logging level.
   sampled_levels:
     'trace/webrequest': 0.2
+  # Set up the limiter for logging. When the rate of logs is close to the limit,
+  # it starts to increase the probability of discarding the logs of same kind.
+  # For default parameters use rate_limit: true
+  rate_limit:
```

Comment:
Both are rather similar in that we are interested in getting a sampled set of data points, with a limited overall rate. The main downside I see in fully replacing static sampling rates with dynamic ones is that it will be slightly harder to reason about the resulting distribution of data points if the underlying event is very bursty.